.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "trailingComma": "es5"
+}
```

Comment:
I would totally be cool with not overriding either of these and just using the default values. For code style, I generally prefer to do as little customization as possible so that the code style is as similar to other projects that use the tool as well. If these conflict with our ESLint rules, I prefer we disable/change the ESLint configuration over changing the Prettier settings, since it's trivial to fix Prettier issues (just run Prettier on the code), while not all ESLint issues have automatic fixes available, since they're not all about code formatting.

What do you think?

Reply:
That is a good point. I think I set these more by force of habit. I have removed the custom configs!